app/views/markets/index.html.erb

```diff
@@ -1,25 +1,24 @@
-<header>
-<div class="col d-flex justify-content-center"><h2>Closest Farmer's Markets<h2></div>
-</header>
-<body>
-<%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-<div class="row">
-  <% @markets.each do |market| %>
-    <div class="col-lg-4 col-md-6 mb-4 d-flex justify-content-center">
-      <div class="my-3 card" style="width:500px">
-        <div style="background-color: rgba(239, 91, 91)">
+<div class="container">
+  <div class="col d-flex justify-content-center"><h2>Closest Farmer's Markets</h2></div>
+  <div class="row">
+    <% @markets.each do |market| %>
+      <div class="col-lg-4 col-md-6 mb-4 d-flex justify-content-center">
+        <div class="card bg-gradient-light my-3" style="min-width:400px">
           <div class="card-body">
-            <div class="d-flex justify-content-center"><%= link_to market.name, market_path(market.id, original_lat: params[:latitude], original_lon: params[:longitude]), method: :get, class: "btn btn-outline-primary me-2 btn-lg" %></div>
+            <div class="d-flex justify-content-center text-center">
+              <%= link_to market.name, market_path(market.id, original_lat: params[:latitude], original_lon: params[:longitude]), method: :get, class: "btn btn-outline-primary me-2 btn-lg" %>
```

Comment:
Were original_lat and original_lon params from when we were using Transit? Are we passing them the same way now that we're using Google?